server/lib/csp.js

```diff
@@ -13,10 +13,15 @@ var helmet = require('helmet');
 var config = require('./configuration');
 var url = require('url');
 
-var SELF = "'self'";
+var SELF = "'self'"; //jshint ignore: line
```

Comment:
@vladikoff Oh yeah, it's pretty aggressive! It's a lot closer to the syntax we use on fxa-auth-server projects (lack of semi-colons, etc).
And thankfully there is a **standard** auto-formatter, [**standard-format**](https://www.npmjs.com/package/standard-format).